src/components/BooksQuery.tsx

```diff
@@ -53,13 +52,14 @@ const BooksQuery = () => {
         <button onClick={submit}>FIND</button>
       </div>
       <div className="books">
-        {list.map((book: book) => (
+        {list.books.map((book: BookObject) => (
           <div key={book.id} className="card-list">
             <Card>
               <CardContent>{book.title}</CardContent>
               <Typography sx={{ mb: 1.5 }} color="text.secondary">
-                {book.author[0]}
-                <p> pupblished in {book.year}</p>
+                {book.author.map((auth) => (
+                  <p key={uuidv4()}>{auth}</p>
```

Comment:
The id will change on every render. This is not how the `key` prop is supposed to be used in React. Please read more about why and how we should use the `key` prop